src/main/java/com/garbagemule/MobArena/commands/setup/AddArenaCommand.java

```diff
@@ -10,9 +10,6 @@
 import org.bukkit.command.CommandSender;
 import org.bukkit.entity.Player;
 
-import java.util.Arrays;
-import java.util.stream.Collectors;
-
```

Comment:
These belong in the unused import commit.

Reply:
ok, I moved this into a new commit